app/src/main/java/io/intrepid/contest/screens/contestcreation/describecontest/DescribeContestFragment.java

```diff
@@ -0,0 +1,54 @@
+package io.intrepid.contest.screens.contestcreation.describecontest;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.text.TextUtils;
+import android.view.View;
+import android.widget.EditText;
+
+import butterknife.BindView;
+import butterknife.OnTextChanged;
+import io.intrepid.contest.R;
+import io.intrepid.contest.screens.contestcreation.BaseNewContestFragment;
+import io.intrepid.contest.screens.contestcreation.NewContestActivity;
+
+public class DescribeContestFragment extends BaseNewContestFragment {
+    private static final String CONTEST_NAME = "ContestName";
+    @BindView(R.id.contest_description_edittext)
```

Comment:
I like to add a blank line between Butterknife variables and the others, not sure if you meant to do the same or if you don't care for it (which is totally fine)